modules/gapi/src/streaming/onevpl/accelerators/accel_policy_cpu.cpp

```diff
@@ -173,8 +178,9 @@ VPLCPUAccelerationPolicy::create_surface_pool(size_t pool_size, size_t surface_s
         GAPI_LOG_INFO(nullptr, "Released workspace memory: " << ptr);
         ptr = nullptr;
 #else
-        GAPI_Assert(false && "Not implemented for systems differ than \"_WIN32\". "
-                             "Please feel free to set it up under OPENCV contribution policy");
+        free(ptr);
```

Comment:
..at least this could have been automated then

Reply:
it's deleter definition for RAII-d shared_ptr, which constructed from raw memory pointer
So, RAII was applied  in result :)